build.gradle.kts

```diff
@@ -407,8 +407,8 @@ subprojects {
     reports.html.outputLocation.set(file("${rootProject.projectDir}/build/reports/"))
     val skipTests = project.hasProperty("skipTests")
     if (!skipTests) {
+      jvmArgs = listOf("-Xmx2G")
```

Comment:
all UTs need 2G JVM memory?

Reply:
The ·-Xmx2G· setting is the maximum; it does not allocate that size immediately, so there should be no problem.